src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/README.md

```diff
@@ -86,10 +86,17 @@ if (err != 0)
 
 try
 {
+    //
+    // During development, it is useful to be able to validate against custom SSL
+    // certificates.  This logic would generally be removed in official releases.
```

Comment:
AddSslCertificate adds an SSL certificate to the collection of in-built official Microsoft intermediate certificates.  Any of the certificates in the collection will satisfy the certificate chain validation, so this is how someone would use a test certificate.  In official releases, only the official Microsoft intermediate certificates are needed, so there is no need to add additional certs.